types/modules/react.d.ts

```diff
@@ -0,0 +1,6 @@
+import type React from 'react';
+
+declare module 'react' {
```

Comment:
Won't this affect repos that use Onyx? Is this wanted?

Reply:
I tested here and it won't affect, btw the .d.ts files won't be on `dist` folder anymore so it won't be copied.